### DISCUSSION TITLE
split, hsplit and vsplit all behave identical

### DISCUSSION THREAD

**Discussion by rasivasu** [kovidgoyal/kitty#4844] [Q&A] [answered]
After adding the following configuration to kitty.conf, save the file and reload the kitty configuration, Pressing "F4", "F5", and "F6" does exactly the same thing.  What it does depends on the windows layout that I select using ⇧⌘l.  I am using macos and btw, ⇧⌃l does not switch the layout.

After splitting the layout, F7 does not do anything other than beeping.  

Please let me know if I have to change any configuration to make the below mapping work.

```
# Create a new window splitting the space used by the existing one so that
# the two windows are placed side by side if the existing window is wide or
# one above the other if the existing window is tall.
map F4 launch --location=split

# Create a new window splitting the space used by the existing one so that
# the two windows are placed one above the other
map F5 launch --location=hsplit

# Create a new window splitting the space used by the existing one so that
# the two windows are placed side by side
map F6 launch --location=vsplit

# Rotate the current split, chaging its split axis from vertical to
# horizontal or vice versa
map F7 layout_action rotate

# Move the active window in the indicated direction
map shift+up move_window up
map shift+left move_window left
map shift+right move_window right
map shift+down move_window down

# Switch focus to the neighboring window in the indicated direction
map shift+ctrl+left neighboring_window left
map shift+ctrl+right neighboring_window right
map shift+ctrl+up neighboring_window up
map shift+ctrl+down neighboring_window down
```

**Comment by kovidgoyal** [comment, marked as answer]
These apply *only* when using the splits layout. So you that layout and
you will be fine.

**Comment by rasivasu** [comment]
Thank you very much.

On Thu, Mar 17, 2022 at 11:35 AM Kovid Goyal ***@***.***>
wrote:

> These apply *only* when using the splits layout. So you that layout and
> you will be fine.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/kovidgoyal/kitty/discussions/4844#discussioncomment-2378172>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIZULXM7HW4W6VT22RW6Q3VALDSTANCNFSM5Q545E4A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Regards,
Ravi
